extras/controllerClient/examples/example_rust/build.rs

```diff
@@ -0,0 +1,43 @@
+// Copyright(C) 2023 NV Access Limited, Leonard de Ruijter
+// This file is covered by the GNU Lesser General Public License, version 2.1.
+// See the file license.txt for more details.

```

Comment:
could you use the more standard python headers? we don't have a license.txt, we use "COPYING"

Reply:
There is a `license.txt` in the `controllerClient` folder.

Reply:
Thanks @lukaszgo1, I wasn't aware of these guidelines on the wiki.